Runtimes/Supplemental/CMakeLists.txt

```
@@ -22,6 +22,7 @@ endif()
 
 set(COMMON_OPTIONS
   -DBUILD_SHARED_LIBS=${BUILD_SHARED_LIBS}
+  -DSDKROOT=${SDKROOT}
```

I believe we want to pass `Swift_SDKROOT` to be used by `find_package(SwiftCore...)` -- `SDKROOT` is meant to be an environment variable, and that should be passed as a result of child processing inheriting the environment of the parent.

```suggestion
  -DSwift_SDKROOT=${Swift_SDKROOT}
```